index.html

```diff
@@ -7,70 +7,412 @@
+                                    <p class="tag bg-light-green">Ruby on rails</p>
+                                </li>
+                                <li>
+                                    <p class="tag bg-light-green">css</p>
+                                </li>
+                                <li>
+                                    <p class="tag bg-light-green">JavaScript</p>
+                                </li>
+                                <li>
+                                    <p class="tag bg-light-green">html</p>
+                                </li>
+                            </ul>
+                            <a class="button bg-green white card" href="#">See Project</a>
+                        </section>
+
+                    </section>
+                </section>
+
+            </section>
+        </section>
+        <section class="about-me">
+            <section class="centered-column bg">
+
+                <header >
+                    <h2 class="dark-blue">
+                        About Me
+                    </h2>
+                    <section class="center">
+
+                        <h4 class="blue ">
+                            I can help you build a product , feature or website Look through some of my work and experience!
+                            If you like what you see and have a project you need coded,
+                            don’t hestiate to contact me.
+                        </h4>
+                    </section>
+                   
+                </header>
+                <a class="button bg-green white card" href="#"> Get My Resume</a>
+            </section>
+            <section class="bordered-card centered-column
+                 bg-light-green">
+                <img class="illustration" src="../assets/icons/about/icon-languages.svg" alt="globe">
+                <h3 class="neutal-blue">Languages</h3>
+                <ul class="tag-container">
+                    <li>
+                        <p class="tag bg-white green">
+                            JavaScript
+                        </p>
+                    </li>
+                    <li>
+                        <p class="tag bg-white green">
+                            Ruby
+                        </p>
+                    </li>
+                    <li>
+                        <p class="tag bg-white green">
+                            Html
+                        </p>
+                    </li>
+                    <li>
+                        <p class="tag bg-white green">
+                            Css
+                        </p>
+                    </li>
+                </ul>
+            </section>
+            <section class="bordered-card centered-column bg-light-green">
+                <img class="illustration" src="../assets/icons/about/icon-frameworks.svg" alt="cubes">
+                <h3 class="neutal-blue">Frameworks</h3>
+                <ul class="tag-container">
```

Comment:
- [ ] The image for the **_languages_** section in the **_About Me_** section is also broken. I think you should check the file path used as the `src` for the image and fix it. 🙏 

## Broken Image

![image](https://user-images.githubusercontent.com/77106962/143587944-b807fdb7-2d62-45fa-8f5e-05b4a4499883.png)